README.md

```diff
@@ -219,7 +223,60 @@ sources:
 
 - Wikipedia, the free encyclopedia (http://www.wikipedia.org)
 
+It contains data from various SUBTLEX word lists: SUBTLEX-US, SUBTLEX-UK, and
+SUBTLEX-CH, created by Marc Brysbaert et al. and available at
+http://crr.ugent.be/programs-data/subtitle-frequencies. SUBTLEX was first
+published in this paper:
+
+
+I (Rob Speer) have
```

Comment:
That whole paragraph of the README is kind of messed up, but fixed in the next PR.